.azure/templates/jobs/build/release_artifacts.yaml

```diff
@@ -1,6 +1,12 @@
 jobs:
   - job: 'release_artifacts'
     displayName: 'Prepare release artifacts'
+    # Strategy for the job
+    strategy:
+      matrix:
+        'deploy_strimzi_java-17':
+          jdk_version: '17'
+          main_build: 'true'
```

Comment:
I do not think you want to have any matrix here. This will never be needed to run on multiple Java versions as you will release only from one of them.

Reply:
Yeah good point, thanks. I removed this with the `JDK_VERSION` parameter, so the default is used from the `install_java.yaml`.
I wanted to keep it consistent with other things, but I totally agree that it doesn't make sense. Thanks